Objects/longobject.c

```diff
@@ -5852,6 +5860,13 @@ int_from_bytes_impl(PyTypeObject *type, PyObject *bytes_obj,
     return long_obj;
 }
 
+void
+_PyLong_ExactDealloc(PyObject *op)
```

Comment:
My goal was to make any future freelist implementation a bit easier, but that sounds like it would be just as easy to do when/if we do that, so I'll drop it.